docs/client.md

```diff
@@ -0,0 +1,47 @@
+## Slack API Client
+
+### Instantiating the Slack API Client
+
+To instantiate the Slack API Client, use the top level `SlackAPI` export. `SlackAPI` accepts two arguments:
+- `token`: Your application's access token
```

Comment:
Where should users get their token from?

Reply:
It's passed in as part of the Function Context. I omitted that part since technically this lib doesn't _need_ to be used from an App and I didn't want to be too prescriptive